main.ts

```diff
@@ -59,8 +57,8 @@ function createWindow() {
 
   nativeTheme.themeSource = 'light';
 
-  win.setTitle(require('./package.json').name + ' ' + require('./package.json').version);
-  if (serve) {
+  win.setTitle(appTitle);
+  if (isDevMode) {
```

Comment:
I noticed that there is a

```typescript
if (isDevMode) {
  win.webcontents.openDevTools();
}
```

right below this statement, maybe we could combine the two to remove the redundant `if` condition?

Reply:
Yes, we can replace `serve` with `isDevMode`

Reply:
Alright!